libsolidity/ast/ASTJsonImporter.cpp

```diff
@@ -95,6 +95,24 @@ SourceLocation const ASTJsonImporter::createSourceLocation(Json::Value const& _n
 	return solidity::langutil::parseSourceLocation(_node["src"].asString(), m_sourceNames);
 }
 
+vector<SourceLocation> ASTJsonImporter::createSourceLocations(Json::Value const& _node, size_t _size) const
```

Comment:
`_size` parameter and its usage look pretty weird here.  I think it should not be a part of the function. In consequence, the `astAssert` should be moved outside too.